src/App.css

```diff
@@ -0,0 +1,49 @@
+.wrapper {
+  background-color: rgb(30, 6, 94);
+}
+
+body {
+  background-color: rgb(30, 6, 94);
+  margin: auto auto;
+}
+
+.cityLink {
+  color: #fff;
+  display: block;
+  text-align: right;
+  font-size: 18px;
+  text-decoration: none;
+  font-weight: 900;
+}
+
```

Comment:
- [ ] Good job working on the requested change by the previous reviewer. However, as the previous reviewer pointed out, your design is not fitting mobile sizes as there is a horizontal scroll. Kindly make the width of the design fit the width of all mobile screens. I will suggest that when adding width, margins, paddings and height to your design, you should always favor percent-based units (vw, vh, %, rem, em) over the fixed unit (px) as it will improve the responsiveness of your design.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/188131207-4b8bbbf3-a4ba-46e3-96d2-e67de5c9cff6.PNG)

- [ ] Also, kindly fix the header of your design.